nucliadb_core/src/fs_state.rs

```diff
@@ -18,13 +18,11 @@
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
```

Comment:
I think is time to move this module to `nucliadb_vectors`, therefore we change it based on that crates needs. Makes sense for it to be part of the index.